5.0/fr/0x15-V6-Authentication.md

```diff
@@ -0,0 +1,166 @@
+
+## V6.6 Mécanismes d'authentification hors bande
+
+Cela implique généralement que le serveur d'authentification communique avec un appareil physique via un canal secondaire sécurisé. Par exemple, l'envoi de notifications push aux appareils mobiles. Ce type de mécanisme d'authentification est considéré comme « quelque chose que vous possédez ».
+
+Les mécanismes d'authentification hors bande non sécurisés tels que le courrier électronique et la VoIP ne sont pas autorisés. L'authentification PSTN et SMS est actuellement considérée comme des [mécanismes d'authentification « restreints »](https://pages.nist.gov/800-63-FAQ/#q-b01) par le NIST et devrait être déconseillée au profit des mots de passe à usage unique basés sur le temps (TOTP), d'un mécanisme cryptographique ou similaire. La norme NIST SP 800-63B [&sect; 5.1.3.3](https://pages.nist.gov/800-63-3/sp800-63b.html#-5133-authentication-using-the-public-switched-telephone-network) recommande de traiter les risques d'échange d'appareil, de changement de carte SIM, de portage de numéro ou d'autres comportements anormaux, si l'authentification hors bande par téléphone ou SMS doit absolument être prise en charge. Bien que cette section ASVS n'impose pas cela comme une exigence, le fait de ne pas prendre ces précautions pour une application L2 sensible ou une application L3 doit être considéré comme un signal d'alarme important.
+
+Notez que le NIST a également récemment publié des directives [déconseillant l'utilisation des notifications push](https://pages.nist.gov/800-63-4/sp800-63b/authenticators/#fig-3). Bien que cette section ASVS ne le fasse pas, il est important d'être conscient des risques de « push bombing ».
+
+| # | Description | Niveau |
+| :---: | :--- | :---: |
+| **6.6.1** | Vérifier que les mécanismes d'authentification utilisant le réseau téléphonique public commuté (RTPC) pour la transmission de mots de passe à usage unique (OTP) par téléphone ou SMS ne sont proposés que lorsque le numéro de téléphone a été préalablement validé. D'autres méthodes plus robustes (telles que les mots de passe à usage unique à durée déterminée) sont également proposées, et que le service informe les utilisateurs des risques de sécurité qu'elles présentent. Pour les applications de niveau 3, le téléphone et les SMS ne doivent pas être disponibles. | 2 |
+| **6.6.2** | Vérifiez que les demandes d’authentification hors bande, les codes ou les jetons sont liés à la demande d’authentification d’origine pour laquelle ils ont été générés et ne sont pas utilisables pour une demande précédente ou ultérieure. | 2 |
+| **6.6.3** | Vérifiez qu'un mécanisme d'authentification hors bande basé sur du code est protégé contre les attaques par force brute grâce à la limitation du débit. Envisagez également d'utiliser un code avec au moins 64 bits d'entropie. | 2 |
```

Comment:
"limitation du débit" feels like we are limiting the download speed? Can we say "rate limiting"? (https://fr.wikipedia.org/wiki/Rate_limiting)